vermeer-python-client/src/pyvermeer/client/client.py

```
@@ -0,0 +1,63 @@
+from pyvermeer.api.base import BaseModule
+from pyvermeer.api.graph import GraphModule
+from pyvermeer.api.task import TaskModule
+from pyvermeer.utils.log import log
+from pyvermeer.utils.vermeer_config import VermeerConfig
+from pyvermeer.utils.vermeer_requests import VermeerSession
+
+
+class PyVermeerClient:
+    """Vermeer API Client"""
+
+    def __init__(
```

Pylint (3.3.7)

[refactor] Too many arguments (6/5)

(R0913)